packages/callback-hook/hook.js

```diff
@@ -112,6 +112,10 @@ export class Hook {
       }
     }
   }
+
+  each(iterator) {
```

Comment:
if you want to deprecate this it would be nice a jsdoc here saying that.

Reply:
Added, though it is a super soft deprecation and  don't plan on removing this any time soon.